testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestComments.java

```diff
@@ -110,54 +90,52 @@ public void testCommentColumn()
                         "   format = 'RCBINARY'\n" +
                         ")",
                 COMMENT_COLUMN_NAME);
-
         query(createTableSql);
+
+        String createTableSqlPattern = format("\\Q" +
+                        "CREATE TABLE hive.default.%s (\n" +
+                        "   c1 bigint COMMENT 'test comment',\n" +
+                        "   c2 bigint COMMENT '',\n" +
+                        "   c3 bigint\n" +
+                        ")\\E(?s:.*)",
+                COMMENT_COLUMN_NAME);
         QueryResult actualResult = query("SHOW CREATE TABLE " + COMMENT_COLUMN_NAME);
-        assertEquals(actualResult.row(0).get(0), createTableSql);
+        assertThat((String) actualResult.row(0).get(0)).matches(createTableSqlPattern);
```

Comment:
inline `createTableSqlPattern` -- the name is no longer as useful as it used to be.